Documentation/Game Design.txt

```diff
@@ -16,43 +16,77 @@ UI:
   => Map
     - Cursor selects planets
     - Arrow keys control ship
+
   => Current planet info
-    - Displays info when a ship collides with a planet on the map
+    - Displays planet info when a ship collides with a planet on the map
+
   => Selected planet info
-    - Displays info when a player hovers the cursor over a planet
+    - Displays planet info when a player hovers the cursor over a planet
+
   => Dialogue box
     - Displays dialog to player during events
 
 Gameplay:
   => Exploration
-    - Move ship around the map to visit each planet
-    - Movement on the map consumes fuel
+    - Planets
+      > Galaxy contains fixed number of planets
+      > Planets can be undiscovered, discovered, colonized (state machine)
+        + Undiscovered planets can't be selected
+        + Discovered planets become colonized if population is > 0, vice versa
```

Comment:
Vice versa? You mean it becomes uncolonized if population == 0?

Reply:
discovered* my bad

Reply:
Yes